wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsWorkUnitsClient.java

```diff
@@ -382,7 +382,7 @@ public static WUState getStateID(WorkunitInfo wu)
         if (wu != null) {
             if (wu.getStateID() != null) {
                 return getStateID(wu.getStateID());
-            } else if (wu.getState() != null && wu.getState().trim().length() != 0) {
+            } else if (wu.getState() != null) {
```

Comment:
ok, so now it looks like you were only concerned about attempting to translate a string made up of white space, is that no longer a concern? Also, are valid states with leading, trailing white space not a concern? If they are, let's trim in the translate function.